tests/unit_tests/test_wave_06.py

```diff
@@ -106,53 +104,56 @@ def test_swap_best_by_category():
     # Act
     result = tai.swap_best_by_category(
         other_vendor=jesse,
-        my_priority="Clothing",
-        their_priority="Decor"
+        my_priority="Clothing", # tai priority
+        their_priority="Decor"  # jesse priority
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert jesse.inventory == [item_d, item_e, item_c]
```

Comment:
This assertions will fail if the order of items in each list somehow change. The README didn't specify that the order of inventory items needed to be maintained, so we probably don't need to constrain our tests and therefore our functions to these requirements. In fact, if you were to implement the swap that I suggested in a comment using multiple assignment, then these tests could fail because the order of elements would be changed.

While more verbose, you can check that elements are in a list like:
```py
    assert item_d in jesse.inventory
    assert item_e in jesse.inventory
    assert item_c in jesse.inventory
```

And since we're checking Jesse's inventory, it would be make the test more thorough to also check the items in Tai's inventory too.